block_0001/usingmyowntokens-20191030145015.xml

```diff
@@ -1,24 +0,0 @@
-  <date>2019-10-30T14:50:15Z</date>
-  <link>/r/TheMotte/comments/do4gpx/culture_war_roundup_for_the_week_of_october_28/f5szk6c/</link>
-  <title>Political Campaigns Varying by Electorate</title>
-  <body>
-    <div class="md">
-      <p>Hypothesize two worlds.</p>
-      <p>In both worlds, there is universal suffrage. But in world 1, only 1/5th of the population bothers to vote, and in world 2, only 4/5ths of the population bothers to vote. </p>
-      <p>All else equal, I would expect that in world 1, that 1/5th of the voting population is likely to be more educated, more intelligent, wealthier, and higher class. For a bunch of reasons, such as:</p>
-      <ul>
-        <li>You need to have enough free time to educate yourself on the candidates and to take time out of your day to vote</li>
-        <li>The higher class you are, the more power you think you have, and so the more likely you are to exercise your right to vote. </li>
-        <li>The wealthier you are, the higher your expected payout of influencing policy in your favour</li>
-      </ul>
-      <p>etc etc etc. </p>
-      <p>Now, given the above, what differences would you expect to see in the campaigns being run in world 1 vs world 2? One obvious difference I would expect is that, because in world 1 the voting population comes from the higher classes, I would expect that in world 1 the politicians pander more to the higher classes. This is going to manifest in using more complex/analytical/academic words, talking about more abstract ideas, etc</p>
-      <p>Or, to jump out of the analogy and into my point: world 1 is going to look a lot more like the 'long-standing political norms' referenced above, and world 2 is going to look a lot more like Trump.</p>
-      <p>Now, I don't have hard data on this, but it is my subjective impression that for the past, say, fifty years, the united states culture has been very enthusiastic in terms of <em>getting people involved in the electoral process</em>. We've had (nearly) universal suffrage this whole time, but the existence of all these voter engagement efforts suggests that a large chunk of people who could vote, don't. </p>
-      <p>So if we assume that, over the past fifty years, the percentage of the population that engages actively with the political process has increased, then (by the assumptions laid out in my analogy) this must necessarily lower the average level of refinement of the voting population. If this has happened, then in order to win, a politician must intentionally reduce the intellectual level at which they are speaking when they are trying to win votes. </p>
-      <p>Or, in other words: the trend being pointed out is <em><strong>exactly</strong></em> the trend I would expect to see in a world that is extending the franchise to larger and larger groups of people within their society. This isn't an alarming trend, this is a signal of our success at democracy</p>
-    </div>
-  </body>
-</post>
```

Comment:
This person *definitely* did not delete their account, they kinda flamed out (you can't see their removed comments *but there's a lot of them*). However, can you move this to a `deleted` directory instead of deleting the file itself? It'll make it easier for me to track what's going on.

Reply:
Sure. Relatedly, wow, you weren't kidding about the flaming out part.

Reply:
Hah, yep. It happens.